public/JSONtoHTML.coffee

```diff
@@ -0,0 +1,211 @@
+# Variables
+xmlHttp = new XMLHttpRequest
+jsonFile = 'stdLib_base_test.json'
+
+getMainData = (data) ->
```

Comment:
This function is waaay to long. From the code strucutre I can see it has some sections. Each section does something. Please refactor them to separate functions and name them in a way that they describe what each of them is doing.